infra/apim_v2/apim.tf

```diff
@@ -828,9 +828,15 @@ module "apim_billing_portal_v1" {
   api_operation_policies = [
     {
       operation_id = "updateOnboardingRecipientIdUsingPUT"
-      xml_content = templatefile("./api/base_ms_url_policy.xml", {
+      xml_content = templatefile("./api/base_policy_config.xml.tpl", {
```

Comment:
about the comment above, you could use `base_ms_url_policy` for the same result.... using new base_policy_config.xml.tpl are duplicating the using of `<set-header name="Authorization"` because **xml_content** on api_management_api module contains `jwt_base_policy.xml` where it is present.